src/test/java/stepDefinitions/ShopMainPageStepDefinitions.java

```diff
@@ -31,17 +32,41 @@ public void iAmOnShopPageUsingPo() throws Throwable {
 
     @Given("^I should see shop page title (.*)$")
     public void iShouldSeeShopPage(String title) throws Throwable {
-        assertEquals(driver.getTitle(),title);
```

Comment:
Better style is to assign title to a new variable, like titleActual and validate it with provided title from scenario.